sentences/el/_common.yaml

```diff
@@ -177,7 +177,7 @@ lists:
     values:
       - in: "(υγρ(ός|η|ο|οί|ες|α)|ενεργοποιημέν(ος|οι))"
         out: "on"
-      - in: "(ξηρ(ός|η|ο|οι|ες|α)|απενεργοποιημέν(ος|οι))"
+      - in: "(ξηρ(ός|η|ο|οί|ες|α)|απενεργοποιημέν(ος|οι))"
```

Comment:
We should include both versions if different STT engines produce either version

Reply:
change reverted